pyodk/_endpoints/forms.py

```diff
@@ -137,13 +136,8 @@ def update(
                 if not fda.upload(file_path=attach, **fp_ids):
                     raise PyODKError("Form update (attachment upload) failed.")
 
-        # Get a new version - using either a timestamp or the callback.
-        new_version = datetime.now().isoformat()
-        if version_updater is not None:
-            new_version = version_updater(self.get(**fp_ids).version)
+        new_version = datetime.now().isoformat() if definition is None else None
 
-        # Publish the draft.
+        # Publish the draft
```

Comment:
Punctuation is cool

Reply:
😂 I think I was going to change this comment in some way and didn't end up doing so? Will change back.